tools/ci/production-extract-tarball

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+ZULIP_PATH=$(mktemp -d)
+tar -xf /tmp/zulip-server-test.tar.gz -C "$ZULIP_PATH" --strip-components=1
```

Comment:
I squashed this commit with the next commit, because this temporary state has us using different random temporary directories here and in `production-helper`.